backend/apps/events/migrations/0022_auto_20220221_1735.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.2.11 on 2022-02-21 16:35
+
+from django.db import migrations
+
+
+def delete_signup_duplicates(apps, schema_editor):
+    signup_model = apps.get_model("events", "Signup")
+    attendable_model = apps.get_model("events", "Attendable")
+
+    attendables = attendable_model.objects.all()
+
+    for attendable in attendables:
+        event = attendable.event
+        sign_ups = signup_model.objects.filter(event_id=event.id)
+
+        for user_id in sign_ups.values_list("user_id", flat=True).distinct():
+            signup_model.objects.filter(
+                event=event, pk__in=signup_model.objects.filter(user_id=user_id).values_list("id", flat=True)[1:]
+            ).delete()
```

Comment:
Do we not risk deleting sign_ups from users that have signed off an event?
```suggestion
            signup_model.objects.filter(
                is_attending=True, event=event, pk__in=signup_model.objects.filter(user_id=user_id).values_list("id", flat=True)[1:]
            ).delete()
```